packages/flutter/test/material/text_field_test.dart

```diff
@@ -899,13 +905,19 @@ void main() {
   });
 
   testWidgets('Material cursor golden', (WidgetTester tester) async {
-    final Widget widget = overlay(
-      child: const RepaintBoundary(
-        key: ValueKey<int>(1),
-        child: TextField(
-          cursorColor: Colors.blue,
-          cursorWidth: 15,
-          cursorRadius: Radius.circular(3.0),
+    final Widget widget = Theme(
```

Comment:
I'm assuming that the extra Theme overrides are needed to affect the overlay and the overlay's child?

Reply:
No. We don't need Theme at all here. This golden test seems failed whenever `useMaterial3` is true/false, I think it might be another case for M1 chip.